test/e2e_node/system/docker_validator_test.go

```diff
@@ -28,7 +28,7 @@ func TestValidateDockerInfo(t *testing.T) {
 		Reporter: DefaultReporter,
 	}
 	spec := &DockerSpec{
-		Version:     []string{`1\.(9|\d{2,})\..*`},
+		Version:     []string{`1\.(9|\d?[0-2]{2,})\..*`}, // Requires 1.9+
```

Comment:
The regex seems wrong to me. For example things like `1.21`, `1.221` will also match.

If we only want to support 1.9-1.12, `1\.(9|1[0-2])\..*` seems better to me